spec/lib/expired_record_cleaner_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe ExpiredRecordCleaner do
+  describe "fiscal year" do
```

Comment:
I should clarify: the word "test" is probably overloaded here. I meant to say something more like "trial" -- that is, I find it prudent to run a task against a single proposal to verify it all works as expected in production, before running it on hundreds of proposals and generating email to lots of users.

I have added a rake task in 8961fdf to illustrate what I mean.